docs/vocascan-server/installation/pm2/pm2.md

```diff
@@ -6,22 +6,24 @@
    sudo apt install nodejs npm git
    ```
 
-   Check if nodejs version is at least v14.x.x
-   
+   Check if nodejs version is at least v12.x.x
+
    ```bash
    node -v
    ```
 
-   if not, you can install it via the [NodeSource PPA]("https://www.digitalocean.com/community/tutorials/how-to-install-node-js-on-ubuntu-20-04")
-   
+   if not, you can install it via the
+   [NodeSource PPA](https://www.digitalocean.com/community/tutorials/how-to-install-node-js-on-ubuntu-20-04)
+
    Retrieve installation script
 
    ```bash
    cd ~
-   curl -sL https://deb.nodesource.com/setup_14.x -o nodesource_setup.sh
+   curl -sL https://deb.nodesource.com/setup_12.x -o nodesource_setup.sh
```

Comment:
I think we can use 14 for the installation, but vocascan-server also works with at minimum 12. Its not in time to recommend a user installing 12.x.x https://nodejs.org/en/about/releases/